src/components/common/TmSessionExtension.vue

```diff
@@ -1,28 +1,26 @@
 <template>
-  <div class="session">
-    <div class="session-container">
-      <div class="session-header">
-        <a @click="goBack">
-          <i class="material-icons session-back">arrow_back</i>
-        </a>
+  <SessionFrame>
+    <div class="session">
+      <div class="session-container">
         <h2 class="session-title">
-          Use Lunie Chrome extension
+          Use Lunie Browser extension
```

Comment:
Is `Chrome` not more specific for other browser users that may think their browser is supported?

Reply:
Brave 🙃